proto/decentraland/social/friendships/friendships.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+package decentraland.social.friendships;
+
+import "google/protobuf/empty.proto";
+
+// Validation and anti-spam errors
+enum FriendshipErrorCode {
+  TOO_MANY_REQUESTS_SENT = 0;
+  NOT_ENOUGH_TIME_PASSED = 1;
+  BLOCKED_USER = 2;
+  NON_EXISTING_USER = 3;
+  INVALID_REQUEST = 4;
+  UNKNOWN = 5;
+}
+
+enum EventType {
```

Comment:
Are we still using this? Should that be encapsulated inside the `FriendshipEvent`?